game/src/core/systems/VelocitySystem.java

```diff
@@ -55,10 +55,24 @@ public void execute() {
     private void updatePosition(VSData vsd) {
         float newX = vsd.pc.getPosition().x + vsd.vc.getCurrentXVelocity();
         float newY = vsd.pc.getPosition().y + vsd.vc.getCurrentYVelocity();
+        ProjectileComponent projectileComponent =
+                (ProjectileComponent) vsd.e.getComponent(ProjectileComponent.class).orElse(null);
         Point newPosition = new Point(newX, newY);
         if (Game.currentLevel.getTileAt(newPosition.toCoordinate()).isAccessible()) {
```

Comment:
@AMatutat Noch drei Ideen/Wünsche für die Game-API (finde das Ticket in der GH-App aufm Cellphone grad nicht):

1. Es gibt immer nur das aktuelle Level, oder? D.h. Abfrage des aktuellen Levels könnte man sich von außen sparen: ein Shortcut `Game.tileAt()` statt `Game.currentLevel.getTileAt()` wäre hilfreich.
2. Die explizite Konvertierung von Point nach Coordinate auf User-Ebene sollte überflüssig gemacht werden und eine überladene Methode `Game.tileAt()` für Point angeboten werden: `Game.tileAt(newPosition).isAccessible()`.
3. Die Benamung und die Nutzung von Attributen vs. Gettern sind uneinheitlich: mal wird direkt der Attribut-Name benutzt, mal eine "get..."-Methode. Entweder geht bitte alles über Methodenaufrufe oder alles per direktem Zugriff auf Attribute, aber bitte kein Mix. Außerdem könnte man bei der Gelegenheit das olle Java-"get..."-Geraffel mal über die Planke gehen lassen und dem Record-Klassenschema folgen: "attributname()" für die getter...

Reply:
1. Hab ich ins richtige Issue/PR gepackt.
2. Jop, siehe auch #547 
3. ack #714